app/models/proposal.rb

```diff
@@ -203,9 +203,6 @@ def delegate_with_default(method)
     end
   end
 
-
-  ## delegated methods ##
-
```

Comment:
I removed this because it seems out of sync with the rest of the app - we aren't using comments to organize methods (and I would argue that we shouldn't have to if we have nice lean models!). But I can add back in.